app/views/school/devices/cannot_order.html.erb

```diff
@@ -3,7 +3,7 @@
 <%- content_for :before_content do %>
   <% breadcrumbs([
     { 'Home' => school_home_path },
-    'Order devices',
+    title,
```

Comment:
The breadcrumb here is intentionally "Order devices" to match the link that was clicked from the homepage. It's for orientation.

Reply:
This has been changed back. I picked it up from here https://github.com/DFE-Digital/get-help-with-tech-prototype/pull/14